src/main/java/com/google/javascript/clutz/ClutzErrorManager.java

```
@@ -0,0 +1,49 @@
+package com.google.javascript.clutz;
+
+import com.google.javascript.jscomp.CheckLevel;
+import com.google.javascript.jscomp.JSError;
+import com.google.javascript.jscomp.MessageFormatter;
+import com.google.javascript.jscomp.PrintStreamErrorManager;
+
+import java.io.PrintStream;
+
+/**
+ * An error manager that delays reporting though {@link #doGenerateReport()}, allows failing fast,
+ * and that adds special, Clutz-specific errors.
+ */
+final class ClutzErrorManager extends PrintStreamErrorManager {
+  private final boolean debug;
+  boolean reportClutzMissingTypes = true;
+
+  /**
+   * @param failEarly if true, directly fail on the first reported error (but not warning!).
```

no such param? also looks like fail early didnt make the cut.